prac_07/project_management.py

```diff
@@ -1,22 +1,30 @@
 """Project Management
 Estimated: 70
-Actual: 68 + 15
+Actual: 118
 """
 import datetime
+import json
+# import pickle
 from operator import attrgetter
 
 from prac_07.project import Project
 
 FILENAME = "projects.txt"
-MENU = """- (L)oad projects  
-- (S)ave projects  
-- (D)isplay projects  
+HEADER = "Name	Start Date	Priority	Cost Estimate	Completion Percentage"
```

Comment:
Could possibly make a function to read and then save this from the file instead of making a constant.